addon/components/radio-select.js

```diff
@@ -1,9 +1,12 @@
-import Component    from 'ember-component';
-import invokeAction from 'ember-invoke-action';
-import computed     from 'ember-computed';
-import get          from 'ember-metal/get';
+import Component        from 'ember-component';
+import invokeAction     from 'ember-invoke-action';
+import computed         from 'ember-computed';
+import get              from 'ember-metal/get';
```

Comment:
There is an empty line between the blocks, so you don't have to align the from to the formFieldOptions